python/cudf/cudf/_lib/utils.pyx

```diff
@@ -311,6 +311,24 @@ cdef data_from_unique_ptr(
     }
     return data, index
 
+cdef columns_from_table_view(
```

Comment:
Did we just not need this before because all the APIs changed up to now generate individual columns and not tables?

Reply:
No, it's because slice interface returns a view of inputs, not owning data. Previous use case uses `columns_from_unique_ptr`.

Reply:
Ah I see. I hadn't anticipated that divergence.